Model/QueuedTask.php

```diff
@@ -160,7 +165,23 @@ public function requestJob($capabilities, $group = null) {
 		//debug($key);ob_flush();
 
 		// try to update one of the found tasks with the key of this worker.
-		$this->query('UPDATE ' . $this->tablePrefix . $this->table . ' SET workerkey = "' . $key . '", fetched = "' . date('Y-m-d H:i:s') . '" WHERE ' . implode(' OR ', $whereClause) . ' ORDER BY ' . $this->virtualFields['age'] . ' ASC, id ASC LIMIT 1');
```

Comment:
remove this line and replace to "$find_jobs_to_update". In postgresql not work because the schema need double quotes and, in postgresql, not have LIMIT in UPDATE command.